server/build.gradle

```diff
@@ -38,7 +38,11 @@ dependencies {
     implementation group: 'io.prometheus', name: 'simpleclient_dropwizard', version: '0.11.0'
     implementation group: 'io.prometheus', name: 'simpleclient_servlet', version: '0.11.0'
     implementation group: 'org.eclipse.jetty', name: 'jetty-servlet', version: '9.4.43.v20210629'
-    testImplementation project(':core').sourceSets.integrationTest.output
+    testImplementation group: 'junit', name: 'junit', version: '4.12'
+    testImplementation group: 'org.assertj', name: 'assertj-core', version: '3.9.1'
+    testImplementation group: 'org.mockito', name: 'mockito-core', version: '2.16.0'
+    testImplementation group: 'org.mockito', name: 'mockito-inline', version: '2.16.0'
+    integrationTestScalarDbServerImplementation project(':core').sourceSets.integrationTest.output
```

Comment:
Just a question for my future update for the ledger project.
Any reason for using `integrationTestScalarDbServerImplementation` instead of `testImplementation`?